packages/x-charts-pro/src/BarChartPro/BarChartPro.tsx

```diff
@@ -29,6 +30,7 @@ export interface BarChartProProps extends BarChartProps {}
  * - [BarChart API](https://mui.com/x/api/charts/bar-chart/)
  */
 const BarChartPro = React.forwardRef(function BarChartPro(props: BarChartProProps, ref) {
+  const { zoom, onZoomChange, ...restProps } = props;
```

Comment:
Reference to recent PR
```suggestion
  const { zoom, onZoomChange, ...other } = props;
```